client/src/pages/Details.js

```diff
@@ -71,14 +76,15 @@ function Details() {
           {filteredMenuItems.map((data) => {
             return (
               <div key={data.id} style={{ display: "flex" }}>
+                <img src={getCloudinaryURL} alt="cloudinary" />
```

Comment:
I assume this use of a functional reference does not work as expected, correct?  

Did you mean to _call_ the function here?  

```suggestion
                <img src={getCloudinaryURL(data.img)} alt="cloudinary" />
```

Reply:
Sounds good.  Let me know when you're ready for another review.